File: BookStore-Angular-MongoDb2/angular/src/app/book/models/book-dto.ts
import { AuditedEntityDto } from '@abp/ng.core';

export class BookDto extends AuditedEntityDto<string> {
  name: string;
  type: any;
  publishDate: string;
  price: number;

  constructor(initialValues: Partial<BookDto> = {}) {
    super(initialValues);
    if (initialValues) {
      for (const key in initialValues) {
        if (initialValues.hasOwnProperty(key)) {
          this[key] = initialValues[key];
        }
      }
    }
  }
}

File: SyncfusionSample/src/SyncfusionSample.Application.Contracts/Books/BookDto.cs
using System;
using Volo.Abp.Application.Dtos;

namespace SyncfusionSample.Books
{
    public class BookDto : AuditedEntityDto<Guid>
    {
        public string Name { get; set; }
        
        public DateTime PublishDate { get; set; }

        public float Price { get; set; }
    }
}